content/blog/2023-03-02-ethereum-classic-course-11-ethereum-classic-social-and-physical-layer-security/index.md

```diff
@@ -0,0 +1,96 @@
+
+## 4. Social Layer: Coordination Problem Security
+
+![The social layer.](./3.png)
+
+On the social layer, the key security component is that all the constituents who participate in the network (i.e. exchanges, wallets, dapps, startups, Node as a Service (NaaS) providers, block explorers, independent participants, mining pools, and independent miners) reside in different parts of the world, in different cultures, with different languages and worldviews making it is extremely difficult to coordinate an attack on the network, or to censor or corrupt it. 
+
+The economic incentives to use the network are conducive to increased use as demand for decentralized computing grows. When demand for decentralized computing grows, more dapp developers build useful things on ETC. The more useful dapps that exist on the blockchain, the more users are attracted to it. As more users are attracted to ETC, then more node operators will participate. The more node operators participating from different parts of the world, the more difficult it becomes to coordinate attacks, making the blockchain even more secure. And when ETC is more secure, the demand for its decentralized computing grows, encouraging even more dapp developers to build on ETC, again continuing this virtuous circle.
+
+## 5. The ETC Security Model Combined
+
+![The ETC security model combined.](./4.png)
+
+The way these two security virtuous circles work combined; with proof of work as the key component at the live operating network, and the coordination problem as the key component at the social layer; is that both have incentives to increase price and participation, and as both sides grow, then demand for ETC computing services grows, which increases price and participation even more.
+
+## 6. Social Layer Attacks and Defense Mechanisms
+
+But what are the possible attacks at the social layer and what are the defense mechanisms?
+
+**Proposals for bad changes:** These are fixes or upgrades that reduce decentralization. Usually these proposals need to go through the Ethereum Classic Improvement Proposal process (ECIP). The way the global ecosystem defends ETC against this type of attack is by opposing and not approving such changes, or not adopting them if for some reason they are approved.
+
+**Censorship:** When Ethereum migrated to proof of stake, it became more than 60% censored by the US Office of Foreign Assets Control (OFAC) nearly immediately because staking pools are centralized. The way ETC prevents this is by having miners join and leave the network whenever they want, wherever they are in the world, taking advantage of the lowest electricity prices possible wherever they may be. This level of decentralization is only possible with the attributes of proof of work explained in previous classes and this is why the PoW mining business is practically impossible to capture.
```

Comment:
The implication of "having miners join and leave" is that miners are being directed to do so by ETC. The proof of work protocol makes this possible but does not include an impetus to the miner to join or leave.